frontend/App.tsx

```diff
@@ -1,10 +1,15 @@
 import { StatusBar } from 'expo-status-bar';
 import { StyleSheet, Text, View } from 'react-native';
-
+import useFetch from './utils/useFetch';
+import data from "./api/endpoint.json"
 export default function App() {
+  const [val, callUrl] = useFetch({url:data.baseApiUrl+data.getRegistrationForm.url,method:data.getRegistrationForm.method[0]});
```

Comment:
We need to specify that getRegistrationForm.method should be an array or not. If not, we should add a key for all available methods. For example

`  "getRegistrationForm": {
    "url": "/registration/form/get",
    "method": {
    "GET": "GET",
  }`